app/routes/signups.js

```diff
@@ -2,75 +2,90 @@
 
 const express = require('express');
 const router = express.Router(); // eslint-disable-line new-cap
-const CampaignNotFoundError = require('../exceptions/CampaignNotFoundError');
+const NotFoundError = require('../exceptions/NotFoundError');
+const UnprocessibleEntityError = require('../exceptions/UnprocessibleEntityError');
 const logger = app.locals.logger;
 
-function successResponse(msg) {
-  const response = {
-    success: {
-      code: 200,
-      message: msg,
-    },
-  };
+function sendResponse(res, code, message) {
+  let type = 'success';
+  if (code > 200) {
+    type = 'error';
+    logger.error(message);
+  }
+
+  const response = {};
+  response[type] = { code, message };
 
-  return response;
+  return res.status(code).send(response);
 }
 
 router.post('/', (req, res) => {
   if (!req.body.id) {
-    res.status(422).send({ error: 'id is required' });
+    return sendResponse(res, 422, 'Missing required id.');
   }
-  const signupID = req.body.id;
+  const signupId = req.body.id;
 
   if (!req.body.source) {
-    res.status(422).send({ error: 'source is required' });
+    return sendResponse(res, 422, 'Missing required source.');
   }
   const source = req.body.source;
 
-  logger.debug(`POST /v1/signups { id:${signupID}, source:${source} }`);
+  logger.info(`POST signup:${signupId} source:${source}`);
 
   if (source === process.env.DS_API_POST_SOURCE) {
-    const msg = `Already sent confirmation for signup ${signupID} with source ${source}.`;
+    const msg = `CampaignBot only sends confirmation when source not equal to ${source}.`;
 
-    return res.send(successResponse(msg));
+    return sendResponse(res, 208, msg);
```

Comment:
@sergii-tkachenko Does Quicksilver blindly post Signups for CampaignBot Campaigns regardless of source? I'm assuming yes -- so returning a 208 vs an error code. If QS was only posting where source != `sms-mobilecommons`, then we would want to return an error code, indicating that check isn't working. It'd be cleaner to do the check to avoid unnecessary API lookup requests, but nothing we need for launch either.